frontend/src/game/play/play.js

```diff
@@ -7,23 +7,27 @@ const config = require('config/config.js');
 const backend = config.servers["backend"];
 
 module.exports = {
-    playCard(rownum, cardindex, target) {
-        const playData = {
-            index: cardindex,
-            row: rownum,
-            target: target,
-        };
-        document.querySelector('.card[index="'+cardindex+'"]').style.background = "yellow";
+    playCard(target) {
+        if (target === undefined &&
+            status.clickedCard.getAttribute("add-power")) {
+            status.clickedCard.style.background = "yellow";    
+        } else {
```

Comment:
I go ahead with the playing only if the card doesn't have `add-power` or has a defined `target`. Otherwise I simply mark the card in yellow.